docs/xcode-project/run-tests.md

```diff
@@ -72,7 +72,7 @@ The maximum number of simulator destinations to test on concurrently.
 ##### `-d, --device=TEST_DEVICES`
 
 
-Test destination description. Either a UDID value of the device, or device name and runtime combination. If runtime is not specified, the latest available runtime for given device name will be chosen. For example "iOS 14.0 iPhone SE (2nd generation)", "iPad Pro (9.7-inch)", "tvOS 14.1 Apple TV 4K (at 1080p)", "Apple TV 4K". If no devices are specified, then the default destination will be chosen (see `xcode-project default-test-destination` for more information about default destination). Multiple arguments
+Test destination description. Either a UDID value of the device, or device name and runtime combination. If runtime is not specified, the latest available runtime for given device name will be chosen. For example "iOS 14.0 iPhone SE (2nd generation)", "iPad Pro (9.7-inch)", "tvOS 14.1 Apple TV 4K (at 1080p)", "Apple TV 4K". Default test destination will be chosen if no devices are specified and test SDK is nottargeting macOS. For macOS tests no destination are specified. (See `xcode-project default-test-destination` for more information about default destination). Multiple arguments
```

Comment:
```suggestion
Test destination description. Either a UDID value of the device, or device name and runtime combination. If runtime is not specified, the latest available runtime for given device name will be chosen. For example "iOS 14.0 iPhone SE (2nd generation)", "iPad Pro (9.7-inch)", "tvOS 14.1 Apple TV 4K (at 1080p)", "Apple TV 4K". Default test destination will be chosen if no devices are specified and test SDK is not targeting macOS. For macOS tests no destination are specified. (See `xcode-project default-test-destination` for more information about default destination). Multiple arguments
```

Reply:
This will be fixed by autogenerated docs.